bin/auto_hck

```diff
@@ -16,7 +17,8 @@ module AutoHCK
   AUTOHCK_RETRIES = 5
 
   at_exit do
-    @engine&.close
+    @project&.setup_manager&.close
+    @project&.engine&.close
```

Comment:
if engine is initialized in the scope of project and setup_manager is initialized in the scope of engine, I think it is better to let project and engine handle the closing accordingly.

Reply:
Yes, you a right. Will fix this

Reply:
@Basils Any more comments?